include/swift/SIL/SILOpenedArchetypesTracker.h

```diff
@@ -20,6 +20,12 @@
 
 namespace swift {
 
+// Disable MSVC warning: multiple copy constructors specified.
+#if defined(_MSC_VER)
+#pragma warning(push)
+#pragma warning(disable: 4521)
+#endif
```

Comment:
I was referring to the not-uncommon situation where there's a formal copy constructor and a generic constructor that could also be called with a `const T &` argument, thereby also potentially fulfilling the role of copy constructor.